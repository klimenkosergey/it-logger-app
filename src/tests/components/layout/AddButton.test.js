import React from 'react';
import { shallow } from 'enzyme';

import AddButton from '../../../components/layout/AddButton';

it('should render AddButton correctly', () => {
  const wrapper = shallow(<AddButton />);
  expect(wrapper).toMatchSnapshot();
});
