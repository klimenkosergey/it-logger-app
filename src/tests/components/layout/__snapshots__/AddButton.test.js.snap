// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AddButton correctly 1`] = `
<div
  className="fixed-action-btn"
>
  <a
    className="btn-floating btn-large blue modal-trigger"
    href="#add-log-modal"
  >
    <i
      className="large material-icons"
    >
      add
    </i>
  </a>
  <ul>
    <li>
      <a
        className="btn-floating green modal-trigger"
        href="#tech-list-modal"
      >
        <i
          className="material-icons"
        >
          person
        </i>
      </a>
    </li>
    <li>
      <a
        className="btn-floating red modal-trigger"
        href="#add-tech-modal"
      >
        <i
          className="material-icons"
        >
          person_add
        </i>
      </a>
    </li>
  </ul>
</div>
`;
